### DISCUSSION TITLE
[@types/lru-cache] maxAge is deprecated

### DISCUSSION THREAD

**Discussion by wireta** [DefinitelyTyped/DefinitelyTyped#58671] [Issues with a @types package] [answered]
The initialization option `maxAge` is deprecated, and `ttl` should be used instead.
the `ttl` option is also missing on the library

Here's an error message I got using the latest version lru-cache:
[LRU_CACHE_OPTION_maxAge] DeprecationWarning: The maxAge option is deprecated. please use options.ttl instead.

Unrelated note: My account is new and I'm not sure if that makes me look untrustworthy but I made it just to report this issue.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
@wireta please review #58853

**Comment by dhensby** [comment]
fixed by #58707
